savon-multipart.gemspec

```diff
@@ -16,7 +16,7 @@ Gem::Specification.new do |s|
   s.rubyforge_project = s.name
 
   s.add_dependency "savon", "~> 2"
-  s.add_dependency "mail", "~> 2.6"
+  s.add_dependency "mail", "~> 2"
```

Comment:
If need be I can change this to, `">= 2.6", "< 3"`. I'm not quite sure what the intent is here though.

Reply:
@kruppel I think your proposal is a good idea, mail 2.5.4 has security issues so is better to allow higher versions.